src/cli.ts

```diff
@@ -189,7 +189,7 @@ function getOptions(program: Command, options: OptionValues): OptionValues {
   return { ...allOptions, network }
 }
 
-function capFeeAt(cap: number, usedFee?: string, specifiedFee?: string): void {
+export function capFeeAt(cap: number, usedFee?: string, specifiedFee?: string): void {

```

Comment:
since the function is export please add function description in comment as documentation .